dataland-frontend/src/router/index.ts

```diff
@@ -127,6 +129,11 @@ const routes = [
   },
   {
     path: "/terms",
```

Comment:
change to terms-en

Reply:
Single page is now used for both. Resolving.